datadog/resource_datadog_child_organization.go

```diff
@@ -0,0 +1,112 @@
+package datadog
+
+import (
+	datadogV1 "github.com/DataDog/datadog-api-client-go/api/v1/datadog"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+)
+
+func resourceDatadogChildOrganisation() *schema.Resource {
+	return &schema.Resource{
+		Description: "Provides a Datadog Organization resource. This can be used to create and manage Datadog organizations.",
+		Create:      resourceDatadogChildOrganizationCreate,
+		Read:        resourceDatadogChildOrganizationRead,
+		Update:      resourceDatadogChildOrganizationUpdate,
+		Delete:      resourceDatadogChildOrganizationDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceDatadogOrganizationImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Description:  "The name of the organization",
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringLenBetween(1, 32),
+			},
+			"api_key": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "",
```

Comment:
Each schema should have a description. Feel free to reference the model descriptions in the API docs: https://docs.datadoghq.com/api/latest/organizations/#create-a-child-organization